modules/otel-collector-components.adoc

```diff
@@ -82,6 +82,55 @@ The Jaeger receiver ingests traces in the Jaeger formats.
 <4> The Jaeger Thrift Binary endpoint. If omitted, the default `+0.0.0.0:6832+` is used.
 <5> The  server-side TLS configuration. See the OTLP receiver configuration section for more details.
 
+[id="kubeletstats-receiver_{context}"]
+=== Kubelet Stats Receiver
+
+:FeatureName: The Kubelet Stats Receiver
+include::snippets/technology-preview.adoc[]
```

Comment:
Not sure if it is outside the scope of this ticket, but I noticed the subtopics below this one have a link to the Technology Preview blurb  (as opposed to the snippet). I think it isn't consistent. Not sure if they need to be changed or the snippet should be a link to match those below?